src/components/toast/toast.spec.js

```diff
@@ -164,50 +163,28 @@ describe('Toast', () => {
 });
 
 describe('ToastStyle', () => {
-  it('should render correct style based on classic theme', () => {
+  it('should render correct with style based on default theme', () => {
```

Comment:
This line should be `'should render with correct style based on default theme'`.

Reply:
fixed in 78107be